index.js

```diff
@@ -35,7 +35,7 @@ if (!isProduction) {
   app.use(errorhandler());
 }
 
-app.use(require('./routes'));
+app.use(require('./server/routes'));
 
 // / catch 404 and forward to error handler
```

Comment:
// catch 404 and forward to error handler @CEOehis

Reply:
I've fixed and updated this. @aknwosu